### DISCUSSION TITLE
Not able to preview the Question set

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-Knowlg/Community#142] [Bugs]
**Steps:**

- Open the https://staging.sunbirded.or/g
- Login with content creator credential
- Click on profile Icon
- Click on workspace
- Click on question set
- Fill the all mandatory fields then click on create new button
- Create MCQ question 
- Fill the fields of question, and options
- Click on preview
- observe

**Actual result :** Preview is not happening if user create the MCQ question
**Expected result** : Preview should be happen if user create the MCQ question

**Jira Ticket:** [KN-891](https://project-sunbird.atlassian.net/browse/KN-891)

@pallakartheekreddy can you please check, this is a blocker to our regression

CC: @vinukumar-vs @mmanimegalai @ashokreddy1208 @vaivk369 @shubhambansaltarento

**Comment by vaivk369** [comment]
@pallakartheekreddy 

    "@project-sunbird/sunbird-collection-editor": "5.4.3",
    "@project-sunbird/sunbird-quml-player": "5.7.0",
The same packages have been installed on coKreat portal, but there the question previews are working fine. On Ed portal is preview is blank.
Not able to figure out what would go wrong here. Could not find anything in gray log as well.

**Comment by vaivk369** [comment]
The issue is fixed with the work around, https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8848. 

Currently ED portal package has :

"@project-sunbird/sunbird-quml-player": "5.7.0",
"@project-sunbird/sunbird-quml-player-v9": "5.1.5",

The @project-sunbird/sunbird-quml-player-v9 is packaged as a dependency for @project-sunbird/sunbird-video-player-v9. 
So, we have modified the code such that the collection editor would have provider from @project-sunbird/sunbird-quml-player .
The correct fix would be we dont need to have "@project-sunbird/sunbird-quml-player-v9": "5.1.5", as dependency and  @project-sunbird/sunbird-video-player-v9 should use @project-sunbird/sunbird-video-player" only.
CC: @pallakartheekreddy  @vaibhavbhuva
